build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
     ext.architecture_components_version = '2.0.1'
     ext.espresso_version = '3.1.1'
     ext.moz_components_version = '0.31.0'
+    ext.oss_licenses_version = '17.0.0'
```

Comment:
Where does this version come from? Their docs say 16.0.2 so I'm concerned someone may have maliciously uploaded what appears to be a new version on a different service (e.g. jcenter since it's hosted on `google()`) that we're accidentally downloading.

Reply:
Copy and pasted from the https://github.com/mozilla-mobile/firefox-tv/pull/2902.  @dnarcese do you remember where you found 17.0.0?  I'm going to change this to 16.0.2 for now.

Reply:
Probably from the fenix implementation https://github.com/mozilla-mobile/fenix/blob/master/buildSrc/src/main/java/Dependencies.kt#L18

Reply:
@colintheshots Why did you choose to use 17.0.0 for the OSS licenses library?